contracts/contracts/harvest/SuperOETHHarvester.sol

```diff
@@ -0,0 +1,40 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity ^0.8.0;
+
+import { OETHHarvesterSimple, IERC20, IStrategy, SafeERC20 } from "./OETHHarvesterSimple.sol";
+
+contract SuperOETHHarvester is OETHHarvesterSimple {
+    using SafeERC20 for IERC20;
+
+    constructor(address _wrappedNativeToken)
+        OETHHarvesterSimple(_wrappedNativeToken)
+    {}
+
+    /// @inheritdoc OETHHarvesterSimple
+    function _harvestAndTransfer(address _strategy) internal virtual override {
```

Comment:
you could have a `getTokenReceiver(address token) virtual` function defined in `OETHHarvestersimple`: 
```
function _getTokenReceiver(address token) internal returns(address receiver) {
    receiver = token == WETH ? dripper : strategistAddr;
}
 // and the one from SuperOETHHarveter.sol
function _getTokenReceiver(address token) internal returns(address receiver) {
    receiver = strategistAddr;
}
```

This way the `_harvestAndTransfer` function could remain the same in both files.

Reply:
That's a fair point and I thought about it. But we will have to call the `_getTokenReceiver` inside the loop, since we are using this on the mainnet as well, I didn't wanna be lax with gas usage